docs/source/install.rst

```diff
@@ -26,16 +26,15 @@ Set SERVER_CONTEXT_PATH and NIVIO_BASEURL env vars to the path if nivio won't be
 Demo mode
 ---------
 
-In the demo mode Nivio loads sample data for demonstration purposes. Run
+In the demo mode Nivio loads sample data for demonstration purposes. From the `nivio/src/test` directory, run
 
 .. code-block:: bash
 
     DEMO=1 docker-compose up
 
 then point your browser to http://localhost:8080/render/nivio:example/graph.png
 
-There is also a demo project "nivio-demo" in the repository, which starts an nginx to simulate a remote server.
-
+There is also a demo project "nivio-demo" in the repository under `nivio/nivio-demo`, which starts an nginx to simulate a remote server.
```

Comment:
Were those docker-compose.yml files intended for these purposes?

Reply:
I've deleted the docker-compose file under src/test and referenced the one under nivio-demo